build/license-header.txt

```diff
@@ -1,16 +0,0 @@
-Copyright (c) "Neo4j"
-Neo4j Sweden AB [http://neo4j.com]
```

Comment:
Why was this file removed?

Reply:
Because it is no longer needed. The `license/neo4j_apache_v2/header.txt` supplied by the `org.neo4j.build:build-resources` will be used instead.
See: https://github.com/neo4j/neo4j-java-driver/pull/1496/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R472